src/module/model.ts

```diff
@@ -8,6 +9,19 @@ class RarityField extends fields.StringField<Rarity, Rarity, true, false, true>
     }
 }
 
+class ProficiencyRankField extends fields.NumberField<ZeroToFour, ZeroToFour, true, false, true> {
+    constructor() {
+        super({
+            choices: R.mapValues({ ...CONFIG.PF2E.proficiencyLevels } as Record<ZeroToFour, string>, (v) =>
+                game.i18n.localize(v),
+            ),
```

Comment:
Can't localize because the field can be constructed before localization is initialized.

Reply:
So go with localize=true in the proficiency inputs after all like I did initially?

Reply:
Localize them in the `getData` as needed

Reply:
We need them localized in {{formGroup}} and {{formInput}} specifically. They're smart enough to render number fields as selects if a choices exists.

Reply:
They can (and should) be passed to `{{formGroup}}` already localized

Reply:
for that to happen, don't they need to be localized in the data model itself?

Reply:
Like I said, do it in the `getData` method of the sheet.

Reply:
Actually, at least in spanish as is, its working fine (note: labels are new fields so they're not translated yet)?

![image](https://github.com/user-attachments/assets/8939dcfe-7067-4ba2-9d70-7fd20ea549b1)

![image](https://github.com/user-attachments/assets/5698a10d-6516-42a4-8205-6cc8fef46b70)

Reply:
Don't use `localize=true` because it'll double localize.